lib/src/utils/credentials_storage.dart

```diff
@@ -34,18 +36,21 @@ class FlutterSecureCredentialsStorage implements CredentialsStorage {
   }) async {
     await _storage.write(key: _key, value: credentials);
     await _storage.write(key: _expirationKey, value: expirationDate);
+    _loginToken = credentials;
+    _kratosExpirationToken = DateTime.parse(expirationDate);
   }
 
   @override
-  Future<void> clear() {
-    return _storage.delete(key: _key);
+  Future<void> clear() async {
+    await _storage.delete(key: _key);
```

Comment:
We should be clearing both key and exp date here, let's fix this as well (same with cache)